src/Algebra/Graph/Labelled.hs

```diff
@@ -118,7 +118,7 @@ instance (Eq e, Monoid e, Ord a) => T.ToGraph (Graph e a) where
 -- subgraphs.
 -- Extract the adjacency map of a graph.
 toAdjacencyMap :: (Eq e, Monoid e, Ord a) => Graph e a -> AM.AdjacencyMap e a
-toAdjacencyMap = foldg AM.empty AM.vertex AM.connect
+toAdjacencyMap = foldg AM.empty AM.vertex (\e x y -> AM.trimZeroes $ AM.connect e x y)
```

Comment:
I haven't used this module much, but I'm pretty sure this is the right place to make a minimal change that will appropriately trim `zero` out of the adjacency maps produced from this module.